Traceback (most recent call last):
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/acoca/anaconda3/envs/envai-book/lib/python3.8/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
rgb = modis_dataset['20'].hvplot.image(x='longitude', y='latitude', cmap='viridis', data_aspect=1, hover=False, title='Band 20: 3.7 micron')
points = fire_points.hvplot.points('longitude', 'latitude', c='frp', cmap='autumn', logz=True, alpha=0.4)
rgb*points


## Summary

This notebook has demonstrated the use of:
 - The `satpy` package to easily load, plot and regrid satellite data from the MODIS sensor.
 - `hvplot` to interatively visualise wildfire pixels detected from the MODIS sensor.
 - `geopandas` to load, filter, and manipulate historical wildfire pixel data.
------------------

[0;36m  File [0;32m"/var/folders/l8/99_59fvn4bl2szm125grkgqw0000gr/T/ipykernel_16063/625049626.py"[0;36m, line [0;32m8[0m
[0;31m    This notebook has demonstrated the use of:[0m
[0m         ^[0m
[0;31mSyntaxError[0m[0;31m:[0m invalid syntax

SyntaxError: invalid syntax (625049626.py, line 8)

